app/models/pregnancy.rb

```diff
@@ -1,55 +0,0 @@
-# Object representing pregnancy outcomes and status of a patient.
```

Comment:
Yeah, for some reason this didn't get deleted in the Great Mergening, so seemed appropriate to remove it in here.